week-2-practise/level-2/5-indexOf.js

```diff
@@ -5,13 +5,14 @@
 // TIP: Use the .indexOf() method
 
 function remove(arr, valueToRemove) {
-  return; // complete this statement
+  arr.splice(arr.indexOf(valueToRemove), arr.indexOf(valueToRemove));
```

Comment:
Same issue as above. What are the parameters to the splice function according to documentation?